### PR TITLE
"Added 3 questions to question.json file and updated contributors.md"

### PR DESCRIPTION
**Pull request by glitched-sudhanshu**
#48
This change adds 3 questions to question.json file related to Android and updated contributors.md

-glitched-sudhanshu

### CODE REVIEW

File: CONTRIBUTERS.MD [hunk 1]
```diff
@@ -23,4 +23,5 @@
 | Kanupriya Jain     | [@Kanupriya](https://github.com/Kanupriya-Jain)               | 1                |
 | Sparsh Chadha      | [@Sparsh011](https://github.com/Sparsh011)                    | 1                |
 | Scott Evans        | [@scottajevans](https://github.com/scottajevans)              | 1                |
-| pantbibek21        | [@pantbibek21](https://github.com/pantbibek21)                | 1                |
\ No newline at end of file
+| pantbibek21        | [@pantbibek21](https://github.com/pantbibek21)                | 1                |
```
Comment (glitched-sudhanshu) [inline CONTRIBUTERS.MD:26]: Sorry, added this name again by mistake.

File: Languages/Android/questions.json [hunk 2]
```diff
@@ -577,6 +577,33 @@
...
       "shortAns": "It allows foreign applications to execute our app's code",
       "tag": "pending_intent"
+    },
+    {
+      "id": 64,
+      "question": "What is paging in android",
+      "details": "Paging is an Android Jetpack feature that is used to load and display small amounts of data from a remote server. It uses less network bandwidth. ",
+      "questionType": "multi ",
+      "trueOrFalse": false,
+      "shortAns": "It is used to load in the form of small chunks.",
```
Comment (AndroidLion48) [inline Languages/Android/questions.json:587]: May you rephrase your short answer from ("small chunks." to "small amounts of data.")? @glitched-sudhanshu 

Thank you for participation!

**Review by AndroidLion48 — CHANGES_REQUESTED** [review]
rephrase your short answer from ("small chunks." to "small amounts of data.")

**Review by Tc2r1 — APPROVED** [review]
This looks great, would you be interested in doing small work on the android project? @glitched-sudhanshu  please don't forget to star the repo.

**Comment by glitched-sudhanshu** [comment]
Yes, I would like to work on this project.
Can you please elaborate what exactly I will have to do?

Thank you

**Comment by Tc2r1** [comment]
> Yes, I would like to work on this project. Can you please elaborate what exactly I will have to do? Thank you

Hi, so currently the application is a project consisting of a bunch of modules that all build the same application using different frameworks/libraries.  Currently they all build to show an android Quiz by pulling the information from the languages/android folder. 

There are a few different ways you can contribute to the project.
1) Add a screen that allows you to pick different types of quizzes and flesh out the repository layer to handle loading different types of quizes. 

2) Make splash screens for one of the pre-existing modules

3) Copy a template module (or in the case that you want to make the app in something besides native kotlin or android, use the template as reference) and recreate the application in your own image using your own selection of libraries, (Like Hilt for example) 

There is documentation for these options in the original readme for the project. 

To build the project, after you have the project on your local computer, open the `Interview_Prep_App` folder in Android studio  and it will compile the project, then select whichever module you'll work in and go from there. 

All Modules share the resources from the sharedResources Module.  If you have any questions please post them in discussions or an issue. What do you think?

**Comment by glitched-sudhanshu** [comment]
Hii Nudennie,

Yeah sure, I will work on it. Once I go through the project I will update
you on which part of the app I am willing to work on.

Just a request, if you are comfortable, can you please provide me with your
linkedIn or Twitter handle, so that we can communicate there. It's a little
bit tedious for me to communicate over emails.
Hope you understand.
Thanks.

**Comment by Tc2r1** [comment]
@glitched-sudhanshu  I agree it is SO TEDIOUS, unfortunately I dont use twitter or linkedin as often as I should either, could you create an issue ticket and we can communicate there. If you need real help I can setup a zoom call btw.

**Comment by Tc2r1** [comment]
@glitched-sudhanshu  feel free to contact me on google chat / hangouts (nudenniew@gmail.com)

**Pull request merged**